src/js/data-model.jsx

```diff
@@ -1,82 +1,153 @@
-var React = require('react'),
-    _ = require('underscore');
+var React = require('react');
+var _ = require('underscore');
 
 
-var router = require('./router'),
-    facts = require('./facts');
+var CollapsibleText = require('./collapsible-text');
+var Dqa = require('./dqa');
+var Table = require('./table');
+var resources = require('./resources');
```

Comment:
It's my personal preference because the imports are visually indented relative for the first `var`, but it does not really matter. The fact that they are all at the top of file is obvious indication that they are imports.

I noticed you did this [above](https://github.com/PEDSnet/pedsnet-catalogue/pull/1/files#diff-1670314037f63a9dd0420401ead74b8bR1) so that is why I suggested it be consistent.

> I keep forgetting to bump the semicolon down when adding a new line.

This could be alleviated but at the new import as a line above the last one (with the semicolon).